create_framework/01-introduction.rst

```diff
@@ -0,0 +1,139 @@
+Introduction
+============
+
+`Symfony2`_ is a reusable set of standalone, decoupled, and cohesive PHP
+components that solve common web development problems.
+
+Instead of using these low-level components, you can use the ready-to-be-used
+Symfony2 full-stack web framework, which is based on these components... or
+you can create your very own framework. This book is about the latter.
+
+Why would you like to create your own framework?
+------------------------------------------------
+
+Why would you like to create your own framework in the first place? If you
+look around, everybody will tell you that it's a bad thing to reinvent the
+wheel and that you'd better choose an existing framework and forget about
+creating your own altogether. Most of the time, they are right but there are
+a few good reasons to start creating your own framework:
+
+* To learn more about the low level architecture of modern web frameworks in
+  general and about the Symfony2 full-stack framework internals in particular;
+
+* To create a framework tailored to your very specific needs (just be sure
+  first that your needs are really specific);
+
+* To experiment creating a framework for fun (in a learn-and-throw-away
+  approach);
+
+* To refactor an old/existing application that needs a good dose of recent web
+  development best practices;
+
+* To prove the world that you can actually create a framework on your own (...
+  but with little effort).
+
+This tutorial will gently guide you through the creation of a web framework,
+one step at a time. At each step, you will have a fully-working framework that
+you can use as is or as a start for your very own. We will start with simple
+frameworks and more features will be added with time. Eventually, you will have
+a fully-featured full-stack web framework.
+
+And of course, each step will be the occasion to learn more about some of the
+Symfony2 Components.
+
+.. tip::
+
+    If you don't have time to read the whole book, or if you want to get
+    started fast, you can also have a look at `Silex`_, a micro-framework
+    based on the Symfony2 Components. The code is rather slim and it leverages
+    many aspects of the Symfony2 Components.
+
+Many modern web frameworks advertize themselves as being MVC frameworks. We
+won't talk about the MVC pattern as the Symfony2 Components are able to create
```

Comment:
I believe there should be a comma between "MVC pattern" and "as the", as it currently looks like "we talk about X as if it was Y", but instead it should be "we talk about X because of Y"